armada/conf/default.py

```diff
@@ -17,6 +17,17 @@
 from oslo_config import cfg
 
 default_options = [
+
+    cfg.ListOpt(
+        'armada_apply_roles',
+        default=['admin'],
+        help='IDs of approved API access roles.'),
+
+    cfg.StrOpt(
```

Comment:
That makes a lot of sense to me, although I included these options because we use these to generate a default config file. Do you think it would be beneficial to have these options included in a default config file that is generated by Armada?